doc_src/index.hdr.in

```diff
@@ -17,7 +17,199 @@
 
 \section introduction Introduction
 
-This is the documentation for `fish`, the friendly interactive shell. `fish` is a user friendly commandline shell intended mostly for interactive use. A shell is a program used to execute other programs. For the latest information on `fish`, please visit the <a href="https://fishshell.com/">`fish` homepage</a>.
+This is the documentation for *fish* the <b>f</b> riendly <b>i</b> nteractive <b>sh</b> ell.
+
+A shell is a commandline interpreter. It reads text input from the commandline and interpretes it as commands to operating system, see: <a href="#shell">What is a Shell</a>.
```

Comment:
What about this:

"A shell is a user interface to access an operating system's services. It is called shell since it can be seen as an outer layer to the operating systems. 

Shells come in two general shapes: as graphical user interface (GUI) or as command-line interface (CLI). These two types differ on what they prioritize. GUI-shells offer a high level of user comfort, but a low level of access to the operating system. CLI-shells do the opposite. 

fish can be seen as combining the best of both worlds: it is a CLI-shell and as such offers a high level of access to the operating system. But it also aims for a maximium of usuablity under the constraint of remaining a CLI-shell."

Is that better? For the definition of a shell [see wikipedis on Shell_(computing)](https://en.wikipedia.org/wiki/Shell_(computing))

Reply:
Yes, sounds good!